packages/documentation/src/stories/components/datepicker/datepicker.docs.mdx

```diff
@@ -0,0 +1,90 @@
+import { Meta, Source } from '@storybook/blocks';
+import { PostTabs, PostTabHeader, PostTabPanel } from '@swisspost/design-system-components-react';
+import StylesPackageImport from '../../../shared/styles-package-import.mdx';
+import NgbComponentDemoLink from '../../../shared/nb-bootstrap/ngb-component-demo-link.mdx';
+import NgbComponentAlert from '../../../shared/nb-bootstrap/ngb-component-alert.mdx';
+import NgbComponentImport from '../../../shared/nb-bootstrap/ngb-component-import.mdx';
+import NgbComponentInternationalization from '../../../shared/nb-bootstrap/ngb-component-internationalization.mdx';
+import * as datepickerStories from './datepicker.stories';
+import datepickerI18nProvider from './datepicker-i18n-provider.sample?raw';
+import datepickerI18nProviderImport from './datepicker-i18n-provider-import.sample?raw';
+import datepickerFormatter from './datepicker-formatter.sample?raw';
+import datepickerFormatterImport from './datepicker-formatter-import.sample?raw';
+import datepickerFloatingLabel from './datepicker-floating-label.sample.html?raw';
+import datepickerNavigationArrows from './datepicker-navigation-arrows.sample.html?raw';
+import datepickerValidation from './datepicker-validation.sample.html?raw';
+
+<Meta of={datepickerStories} />
+
+<div className="d-flex align-items-center justify-content-between">
+  # Datepicker
+
+  <NgbComponentDemoLink component="datepicker" />
+</div>
+
+<p className="lead">Display dates on a calendar view, allowing selection of a date or date range.</p>
+
+<NgbComponentAlert component="datepicker" src="https://ng-bootstrap.github.io/#/components/datepicker/overview" />
+
+<ul>
+  <li>
+    <a href="#component-import" target="_self">Component Import</a>
+  </li>
+  <li>
+    <a href="#style-imports" target="_self">Style Imports</a>
+  </li>
+  <li>
+    <a href="#internationalization" target="_self">Internationalization</a>
+  </li>
+  <li>
+    <a href="#date-format" target="_self">Date Format</a>
+  </li>
+  <li>
+    <a href="#examples" target="_self">Examples</a>
+  </li>
+</ul>
```

Comment:
Those don't work anymore since https://github.com/swisspost/design-system/pull/2467. But it's not a blocker for this PR.